lib/fake-xhr/index.test.js

```diff
@@ -1,6 +1,18 @@
 "use strict";
 
 var referee = require("referee");
+
+referee.format = function (object) {
```

Comment:
Overriding `referee.format` is not necessary for the tests to run correctly.

Since we have adopted `referee` as a `@sinonjs` project, I would rather improve `referee` for everyone, than have customisations in all projects that you it.

Would you mind removing the override?